amethyst_renderer/examples/sphere.rs

```diff
@@ -70,11 +68,14 @@ fn main() {
             _ => (),
         });
 
-        renderer.draw(&scene, &pipe, delta);
+        renderer.draw(&scene, &pipe, delta)?;
         delta = Instant::now() - start;
     }
+    Ok(())
 }
 
+quick_main!(run);
```

Comment:
In the future we should replace this with main again: https://github.com/rust-lang/rust/issues/43301